content/en/blog/2024/otel-collector-anti-patterns/index.md

```diff
@@ -0,0 +1,184 @@
+[Core](https://github.com/open-telemetry/opentelemetry-collector), and
+[Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib).
+
+The Core distribution is a bare-bones distribution of the Collector for OTel
+developers to develop and test. It contains a base set of
+[extensions](/docs/collector/configuration/#service-extensions),
+[connectors](/docs/collector/configuration/#connectors),
+[receivers](/docs/collector/configuration/#receivers),
+[processors](/docs/collector/configuration/#processors), and
+[exporters](/docs/collector/configuration/#exporters).
+
+The Contrib distribution is for non-OTel developers to experiment and learn. It
+also extends the Core distribution, and includes components created by
+third-parties (including vendors and individual community members), that are
+useful to the OpenTelemetry community at large.
+
+Neither Core nor Contrib alone are meant to be part of your production workload.
+Using just Core by itself is too bare-bones and wouldn’t suit an organization’s
+needs. (Though its components are absolutely needed!) And although many
+OpenTelemetry practitioners, deploy Contrib in their respective organizations,
+it has many components, and you likely won’t need every single exporter,
+receiver, processor, connector, and extension. That would be overkill, and your
+Collector instance ends up needlessly bloated, potentially increasing the attack
+surface.
+
+But how do you pick and choose the components that you need? The answer is to
+build your own distribution, and you can do that using a tool called the
+[OpenTelemetry Collector Builder](/docs/collector/custom-collector/) (OCB). In
+addition, at some point, you may need to create your own custom Collector
+component, such as a processor or exporter. The OCB allows you to integrate your
+custom components AND pick and choose the Contrib components that you need.
+
+It is also worth mentioning that some vendors build their own
+[Collector distributions](/ecosystem/distributions/). These are OTel Collector
+distributions that are curated to Collector components that are specific to that
+vendor. They may be a combination of custom, vendor-developed components, and
+curated Collector Contrib components. Using vendor-specific distributions
+ensures that you are using just the Collector components that you need, again
+reducing overall bloat.
+
+_Bottom line:_ Using the right distribution reduces bloat and allows you to
+include only the Collector components that you need.
+
+### 4- Not updating your Collectors
+
+This one’s short and sweet. Keeping software up-to-date is important, and the
+Collector is no different! By regularly updating the Collector, it allows you to
+stay up-to-date with the latest version so that you can take advantage of new
+features, bug fixes, performance improvements, and security fixes.
+
+### 5- Not using the OpenTelemetry Collector where appropriate
```

Comment:
@theletterf moved this down as number 5 per your suggestion.

Reply:
@jpkrohling how is this title?

Reply:
Love it — thanks!